lib/interactive.dart

```diff
@@ -149,7 +178,7 @@ class Interactive {
 
   html.DivElement _renderChart() {
     var chartWrapper = html.DivElement()..classes = [CHART_WRAPPER_CSS];
-    var chartWidth = _container.offset.width - 2 * CHART_PADDING;
+    var chartWidth = _chartWidth - 4 * CHART_PADDING;
```

Comment:
Could you use parens please so I don't have to remember the precedence table :)

Reply:
Fixed